Packs/MandiantAdvantageThreatIntelligence/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mandiant Advantage Threat Intelligence
+
+- Fixing `limit` argument for `mati-feed-get-indicators`
```

Comment:
When adding release notes please follow the guide lines on this page:
https://xsoar.pan.dev/docs/documentation/release-notes
```suggestion
- Fixed an issue where *limit* argument from ***mati-feed-get-indicators*** command was not parse properly.
```